src/identifiers.md

```diff
@@ -1,19 +1,25 @@
 # Identifiers
 
 > **<sup>Lexer:<sup>**  
-> IDENTIFIER :  
+> IDENTIFIER_OR_KEYWORD :  
 > &nbsp;&nbsp; &nbsp;&nbsp; [`a`-`z` `A`-`Z`]&nbsp;[`a`-`z` `A`-`Z` `0`-`9` `_`]<sup>\*</sup>  
 > &nbsp;&nbsp; | `_` [`a`-`z` `A`-`Z` `0`-`9` `_`]<sup>+</sup>  
+>  
+> IDENTIFIER :  
+> IDENTIFIER_OR_KEYWORD <sub>*Except a [strict] or [reserved] keyword*</sub>
 
 An identifier is any nonempty ASCII string of the following form:
 
 Either
 
-   * The first character is a letter
-   * The remaining characters are alphanumeric or `_`
+* The first character is a letter
+* The remaining characters are alphanumeric or `_`
 
 Or
 
-   * The first character is `_`
-   * The identifier is more than one character, `_` alone is not an identifier
-   * The remaining characters are alphanumeric or `_`
+* The first character is `_`
+* The identifier is more than one character, `_` alone is not an identifier
```

Comment:
I know this was here previously, but can you change the comma to a period and otherwise add periods to these sentences?